segmentation/include/pcl/segmentation/extract_labeled_clusters.h

```diff
@@ -35,155 +35,217 @@
 
 #pragma once
 
+#include <pcl/search/pcl_search.h>
 #include <pcl/pcl_base.h>
-#include <pcl/search/search.h> // for Search
 
-namespace pcl
-{
+namespace pcl {
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+/** \brief Decompose a region of space into clusters based on the Euclidean distance
+ * between points \param[in] cloud the point cloud message \param[in] tree the spatial
+ * locator (e.g., kd-tree) used for nearest neighbors searching \note the tree has to be
+ * created as a spatial locator on \a cloud \param[in] tolerance the spatial cluster
+ * tolerance as a measure in L2 Euclidean space \param[out] labeled_clusters the
+ * resultant clusters containing point indices (as a vector of PointIndices) \param[in]
+ * min_pts_per_cluster minimum number of points that a cluster may contain (default: 1)
+ * \param[in] max_pts_per_cluster maximum number of points that a cluster may contain
+ * (default: max int) \param[in] max_label \ingroup segmentation
+ */
+template <typename PointT>
+PCL_DEPRECATED(1, 14, "Use of max_label is deprecated")
+void extractLabeledEuclideanClusters(
+    const PointCloud<PointT>& cloud,
+    const typename search::Search<PointT>::Ptr& tree,
+    float tolerance,
+    std::vector<std::vector<PointIndices>>& labeled_clusters,
+    unsigned int min_pts_per_cluster,
+    unsigned int max_pts_per_cluster,
+    unsigned int max_label);
```

Comment:
Missing defaults for parameters?

Reply:
This is the deprecated version. If someone is using with a parameter, only then it affects them. Adding a default to the deprecated version will create a conflict between the two versions